modular_skyrat/master_files/code/modules/clothing/under/syndicate.dm

```diff
@@ -0,0 +1,132 @@
+/obj/item/clothing/under/syndicate
+	worn_icon_digi = 'modular_skyrat/master_files/icons/mob/clothing/under/syndicate_digi.dmi' // Anything that was in the syndicate.dmi, should be in the syndicate_digi.dmi
+
+
+/obj/item/clothing/under/syndicate/skyrat
+	icon = 'modular_skyrat/master_files/icons/obj/clothing/under/syndicate.dmi'
+	worn_icon = 'modular_skyrat/master_files/icons/mob/clothing/under/syndicate.dmi'
+	//These are pre-set for ease and reference, as syndie under items SHOULDNT have sensors and should have similar stats; also its better to start with adjust = false
+	has_sensor = NO_SENSORS
+	armor = list(MELEE = 10, BULLET = 0, LASER = 0,ENERGY = 0, BOMB = 0, BIO = 10, FIRE = 50, ACID = 40)
+	can_adjust = FALSE
+
+//Related files:
+// modular_skyrat\modules\Syndie_edits\code\syndie_edits.dm (this has the Overalls and non-Uniforms)
+// modular_skyrat\modules\novaya_ert\code\uniform.dm (NRI uniform(s))
+
+/*
+*	TACTICOOL
+*/
+
+//This is an overwrite, not a fully new item, but still fits best here.
+/obj/item/clothing/under/syndicate/tacticool //Overwrites the 'fake' one. Zero armor, sensors, and default blue. More Balanced to make station-available.
+	name = "tacticool turtleneck"
+	desc = "A snug turtleneck, in fabulous Nanotrasen-blue. Just looking at it makes you want to buy a NT-certifed coffee, go into the office, and -work-."
+	icon = 'modular_skyrat/master_files/icons/obj/clothing/under/syndicate.dmi' //Since its an overwrite it needs new icon linking. Woe.
+	worn_icon = 'modular_skyrat/master_files/icons/mob/clothing/under/syndicate.dmi'
+	icon_state = "tactifool_blue"
+	inhand_icon_state = "b_suit"
+	can_adjust = TRUE
+	has_sensor = HAS_SENSORS
+	armor = list(MELEE = 0, BULLET = 0, LASER = 0,ENERGY = 0, BOMB = 0, BIO = 0, FIRE = 0, ACID = 0)
+	unique_reskin = list("NT Blue" = "tactifool_blue",
+						"Charcoal" = "tactifool"
+						)
```

Comment:
I'm not going to mention it in every suggestion, but you should make those "NT Blue" etc. reskin names into defines, especially since you're using them in other places in your code.
```suggestion
	unique_reskin = list(
		"NT Blue" = "tactifool_blue",
		"Charcoal" = "tactifool",
	)
```